ios/RNWifi.m

```diff
@@ -53,51 +53,74 @@ + (BOOL)requiresMainQueueSetup
 RCT_EXPORT_METHOD(connectToSSID:(NSString*)ssid
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
-    
+
     if (@available(iOS 11.0, *)) {
         NEHotspotConfiguration* configuration = [[NEHotspotConfiguration alloc] initWithSSID:ssid];
-        configuration.joinOnce = true;
-        
+        configuration.joinOnce = false;
```

Comment:
Maybe some users want to use in true, what do you think to turn this to param?

Reply:
I have made that change as a remedy to issue #2 which I was experiencing as well.

Reply:
let me observe how it behaves in Android, to make this decision

Reply:
Hello @JuanSeBestia, any feedback on this PR?

Reply:
@jkaravakis imo these changes are unrelated to the PR and don't belong in it. why not just set them back so the main functionality of the PR can get merged?

Reply:
I agree, will revert back to `joinOnce = true`.